src/pages/main-page/main-page-pure.tsx

```diff
@@ -60,22 +64,24 @@ const MainPage: FC<Props> = ({
             <SearchEntities entities={entities} />
           </section>
           <section className='col-12 col-lg-4'>
-            <HeaderSC.Header>
-              {localization.community.seeLatest}
-              <SC.Link href={`${FDK_COMMUNITY_BASE_URI}`} external>
-                {localization.community.title}
-              </SC.Link>
-            </HeaderSC.Header>
-            {prunedPosts.length > 0 ? (
-              <SC.CommunityPosts>
-                {prunedPosts.slice(0, 3).map(post => (
-                  <Post key={post.pid ?? post.uid} post={post} />
-                ))}
-                <SC.Link href={`${FDK_COMMUNITY_BASE_URI}/recent`} external>
-                  {localization.community.seeMore}
-                </SC.Link>
-              </SC.CommunityPosts>
-            ) : null}
+            <CommunityContainer>
+              <ContainerHeader>
+                {`${localization.community.seeLatest} ${localization.community.title}`}
+              </ContainerHeader>
+              {prunedPosts.length > 0 ? (
+                <SC.CommunityPosts>
+                  {prunedPosts.slice(0, 3).map(post => (
+                    <>
+                      <Post key={post.pid ?? post.uid} post={post} />
+                      <Divider />
```

Comment:
Vanligvis bedre å klare seg uten Divider-komponent. Du har to andre muligheter, enten lage en style på hver Post med border-bottom, eller lage en style på CommunityPosts f.eks.:

& > a { 
 border-bottom: ...
}